vnext/ReactUWP/ReactUWP.vcxproj

```diff
@@ -403,13 +407,18 @@
   <ImportGroup Label="ExtensionTargets">
     <Import Project="$(SolutionDir)packages\boost.1.68.0.0\build\boost.targets" Condition="Exists('$(SolutionDir)packages\boost.1.68.0.0\build\boost.targets')" />
     <Import Project="$(SolutionDir)packages\boost_date_time-vc141.1.68.0.0\build\boost_date_time-vc141.targets" Condition="Exists('$(SolutionDir)packages\boost_date_time-vc141.1.68.0.0\build\boost_date_time-vc141.targets')" />
+    <Import Project="..\packages\vcpkg-export-20190805-175352.1.0.0\build\native\vcpkg-export-20190805-175352.targets" Condition="Exists('..\packages\vcpkg-export-20190805-175352.1.0.0\build\native\vcpkg-export-20190805-175352.targets')" />
```

Comment:
>vcpkg-export-20190805-175352.1.0.0 [](start = 33, length = 34)

This appears to be missing from packages.config; is it still needed?

Reply:
Ah .. thank you for pointing this out.. This was a test hermes nuget that i was using to test earlier.. I'll remove this entry.